jenkins/ml-models.JenkinsFile

```diff
@@ -26,25 +26,49 @@ pipeline {
             description: 'Version number of the model',
             trim: true
         )
-        }
+        choice(
+            name: 'FORMAT',
+            description: 'Tracing format of the model',
+            choices: ['', 'BOTH', 'TORCH_SCRIPT', 'ONNX']
```

Comment:
Why an empty string at the start?

Reply:
- There are two types of updates we want to make: model listing & models.
- The first choice will be the default choice, and we need empty string to be there for the case where we want to update model listing so that we don't need to pass in parameter 'FORMAT' into JENKINS_PARAM. 
- Another way to do this is to allow model listing to be updated given any FORMAT, but I'm not sure if that will make it confusing when we manually trigger the workflow because we will see some choice (i.e., BOTH) as a default.

Reply:
Can we add comment in the code to clarify?

Reply:
Sure!